src/Npgsql/SqlQueryParser.cs

```diff
@@ -3,7 +3,7 @@
 using System.Diagnostics;
 using System.Text;
 
-#if NET461 || NETSTANDARD2_0
+#if NETSTANDARD2_0
```

Comment:
The same here.

Reply:
This one needs to stay because of NETSTANDARD2_0, and it seems cleaner to separate it out of the main block...